src/main/java/uk/gov/companieshouse/accounts/association/models/email/builders/InviteEmailBuilder.java

```diff
@@ -9,7 +11,7 @@ public class InviteEmailBuilder extends EmailBuilder<InviteEmailBuilder, InviteE
 
     private String invitationExpiryTimestamp;
 
-    private String invitationLink;
+    private final String invitationLink = CHS_URL + "/your-companies/company-invitations?mtm_campaign=associations_invite";
```

Comment:
i would move this to properties file and use @value for url.
if this changes in future then we dont need to change the code, we can just update the property values in live.